site/layouts/index.redirects

```diff
@@ -0,0 +1,13 @@
+{{ $versions := site.Params.versions }}
+{{ $latest   := site.Params.latest }}
+/docs                          /docs/{{ $latest }}     301!
+/docs/latest                   /docs/{{ $latest }}
+/docs/latest/*                 /docs/{{ $latest }}/:splat
```

Comment:
This would presumably catch all pages, but in testing out the restic page on the preview, its giving me a 404.

Reply:
It catches all pages after latest, so https://deploy-preview-2895--velero.netlify.app/docs/latest/restic works.

Reply:
Ahhhhhhhh, thanks, I was having a moment :)

Will use that going forward :)